src/pages/ReimbursementAccount/BeneficialOwnersStep.js

```diff
@@ -145,6 +175,12 @@ class BeneficialOwnersStep extends React.Component {
                         onPress={() => {
                             this.setState((prevState) => {
                                 const hasOtherBeneficialOwners = !prevState.hasOtherBeneficialOwners;
+                                this.debouncedUpdateReimbursementAccountDraft({
+                                    hasOtherBeneficialOwners: !prevState.hasOtherBeneficialOwners,
+                                    beneficialOwners: hasOtherBeneficialOwners && _.isEmpty(prevState.beneficialOwners)
+                                        ? [{}]
```

Comment:
Is it necessary to set the empty object inside the empty array? In the `submit` function we are falling back to an empty array, so just wondering why the empty object is necessary here.

Reply:
The empty object is necessary in this case. We want at least one item in the array in order to render the form when we call `_.map` [here](https://github.com/Expensify/App/blob/main/src/pages/ReimbursementAccount/BeneficialOwnersStep.js#L165)

The reason for the empty array in the submit function is explained [here](https://github.com/Expensify/App/blob/main/src/pages/ReimbursementAccount/BeneficialOwnersStep.js#L103-L106)